src/ListItem.js

```diff
@@ -0,0 +1,6 @@
+function ListItem(props){
+  return (
+    <li key={props.id}>{props.title}</li>
```

Comment:
When we assign keys, we want to do it inside of the context of the array we are `map`ping over. So, instead of placing this inside of your `ListItem.js` file, we want the key to go on the implementation of `ListItem` in your `TodoList.js` file

See: [Extracting Components with Keys](https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys)

> "_Keys only make sense in the context of the surrounding array...A good rule of thumb is that elements inside the map() call need keys._"

Reply:
Thank you for taking time finding the resource for me. I understand it better now!
> A good rule of thumb is that elements inside the map() call need keys.